streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBTimestampedWindowStore.java

```diff
@@ -16,21 +16,14 @@
  */
 package org.apache.kafka.streams.state.internals;
 
-import java.nio.ByteBuffer;
+import org.apache.kafka.streams.state.TimestampedBytesStore;
 
-import static org.apache.kafka.clients.consumer.ConsumerRecord.NO_TIMESTAMP;
+class RocksDBTimestampedWindowStore extends RocksDBWindowStore implements TimestampedBytesStore {
 
-class StoreProxyUtils {
```

Comment:
This whole diff is a little weird.
 - `StoreProxyUtils` is not needed any longer an removed
 - we add a new `RocksDBTimestampedWindowStore` (need this to implement the marker interface)

No idea why we see a diff -- both are not related. This should rather be an independent add and remove.